test/unit/auth/auth.spec.ts

```diff
@@ -166,6 +166,29 @@ function getDecodedIdToken(uid: string, authTime: Date, tenantId?: string): Deco
   };
 }
 
+/**
+ * Generates a mock decoded ID token with the provided parameters.
+ *
+ * @param {string} uid The uid corresponding to the ID token.
+ * @param {Date} authTime The authentication time of the ID token.
+ * @param {string=} tenantId The optional tenant ID.
+ * @return {DecodedIdToken} The generated decoded ID token.
```

Comment:
Types in curly braces are not needed for API Extractor.

Reply:
removed